index.html

```diff
@@ -216,6 +216,32 @@ <h1 class="about-title">About <br>Myself</h1>
       </div>
    </section>
 
+<!--Here begins the Contact form-->
+<footer>
+   <div class="contact-wrapper" id="contact">
+      <div class="contact-section">
+         <h1 class="contact-title">Contact me</h1>
+         <p class="contact-desc">
+            If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
+         </p>
+         <form action="https://formspree.io/f/xbjbldyw" method="post">
+            <div class="form-items">
+               <label for="name">
+                  <input type="text" id="name" name="name"placeholder="User name" maxlength="30" required>
+               </label>
+               <label for="mail">
+                  <input type="email" id="mail" name="email" placeholder="User email Address" required>
+               </label>
+               <textarea name="user_msg" id="message"name="message" cols="30" rows="10" maxlength="500" placeholder="Write your message here" required></textarea>
+               <button type="submit" class="contact-btn">Get in Touch</button>
+            </div>
+         </form>
+      </div>
+
+      </div>
+   </footer>
 </body>
```

Comment:
- [ ] Kindly as stated by the previous reviewer 😃, Kindly compare both design and fix the position of the icons in background images to match the given design, as you see here most of the background image of this footer is hidden, so please try to place it at the correct position to stick to the Figma Design 💯🚀

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Original design | Your design| 
| --- | --- |
|  ![image](https://github.com/cloud007bond/Portfolio/assets/83360636/b35636dd-1688-4af2-a640-811c471d8c4a)  |    ![image](https://github.com/cloud007bond/Portfolio/assets/83360636/3a528d63-489c-4fd3-9b9c-e98223cee84a) |



| Figma design | Your design| 
| --- | --- |
|    ![image](https://github.com/cloud007bond/Portfolio/assets/83360636/8bd19c7b-2d14-40cd-90ec-cabb4067b861)  |   ![image](https://github.com/cloud007bond/Portfolio/assets/83360636/4ba91cc6-207b-429e-9098-6be3c82028dd)    |